llms/mlx_lm/models/su_rope.py

```diff
@@ -0,0 +1,70 @@
+import math
+import mlx.core as mx
+
+
+class Phi3SuScaledRotaryEmbedding:
```

Comment:
I'm wondering if it's possible to implement this in terms of `mx.fast.rope` as it will be much much faster. It looks to me like you can pass in the appropriate base and scaling factor and achieve that?